__tests__/queries.js

```diff
@@ -57,6 +57,16 @@ test.skip('Transaction List', async () => {
     // );
 });
 
+test('Block signatures', async () => {
```

Comment:
Please disable the test for nodeSE as non-valid

Reply:
fixed

Reply:
fixed